.github/workflows/ci.yaml

```diff
@@ -7,6 +7,13 @@ on:
   pull_request:
 
 jobs:
+  buf-lint:
```

Comment:
buf-breaking doesn't work until we have proto files in the repo :(
So once this lands then I can add the buf-breaking ci check